odd-platform-ui/build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
 node {
     download = true
     version = "16.14.0"
+    npmInstallCommand = "ci"
```

Comment:
@DementevNikita it didn't work, did it?

Reply:
that changes `npm install` to the `npm ci` which is meant to be used in a CI environment.